src/main/java/hw_2/Task1.java

```diff
@@ -0,0 +1,77 @@
+package hw_2;
+
+public class Task1 {
+    public static void main(String[] args) {
+        // +
+        int a = 10;
+        int b = 5;
+        int c = a+b;
+        System.out.printf("%d+%d=%d\n", a, b, c);
+        // -
+        a = 20;
+        b = 3;
+        c = a-b;
+        System.out.printf("%d-%d=%d\n", a, b, c);
+        // *
+        a = 2;
+        b = 2;
+        c = a*b;
+        System.out.printf("%d*%d=%d\n", a, b, c);
+        // /
+        a = 10;
+        b = 5;
+        c = a/b;
+        System.out.printf("%d/%d=%d\n", a, b, c);
+        double d = 9.5/5;
+        System.out.printf("9.5/5=%.2f\n", d);
+        // %
+        a = 16;
+        b = 3;
+        c = a%b;
+        System.out.printf("%d%%%d=%d\n", a, b, c);
```

Comment:
Do we really need "%%" between %d and %d

Reply:
Fixed